src/sidebar/components/version-info.js

```diff
@@ -6,8 +6,20 @@ import { withServices } from '../util/service-context';
 
 import Button from './button';
 
+// Type checking imports
+// eslint-disable-next-line no-unused-vars
+import VersionData from '../util/version-data';
```

Comment:
I may have forgotten the right way to do this, but I want to enforce the class type here.

Reply:
Thats what I tried first and I don't think that worked the way I expected it to.. I'll double check. But I may have looked it wrong.

Reply:
Ah, I know the confusion here. You need to explicitly use the name "default" for the default export - `import(".../version-data").default`